.github/workflows/features.yml

```diff
@@ -51,10 +54,13 @@ jobs:
           - requeue
           - runtime
           - server
+          - "server rustls-tls"
+          - "server boring-tls"
+          - "server boring-tls rustls-tls"
           - shutdown
-    timeout-minutes: 5
+    timeout-minutes: 10
     runs-on: ubuntu-latest
-    container: ghcr.io/linkerd/dev:v39-rust
+    container: ghcr.io/linkerd/dev:v40-rust
```

Comment:
Eh, if we're going to bump this, we need to do so everywhere. This should be done in a separate PR #171